docker/docker-compose.yml

```diff
@@ -180,9 +179,44 @@ services:
       localstack:
         condition: service_healthy
 
+  management-ui:
+    healthcheck:
+      test: ["CMD", "wget", "-qo", "/dev/null", "http://localhost:8099/health"]
+      interval: 10s
+      timeout: 10s
+      retries: 3
+    image: quay.io/ukhomeofficedigital/hocs-management-ui:${MANAGEMENT_UI_TAG:-latest}
+    ports:
+      - 8099:8099
+    networks:
+      - hocs-network
+    environment:
+      USE_CLIENTSIDE: "true"
+      WORKFLOW_SERVICE: http://workflow:8080
+      CASEWORK_SERVICE: http://casework:8080
+      INFO_SERVICE: http://info:8080
+      DOCUMENT_SERVICE: http://documents:8080
+      ALLOWED_FILE_EXTENSIONS: "doc,docx,txt,rtf,html,pdf,jpg,jpeg,tif,tiff,png,bmp,gif,xls,xlsx,msg"
+      S3_BUCKET: "untrusted-bucket"
+      S3_ENDPOINT: http://localstack:4572
+      DEFAULT_TIMEOUT_SECONDS: 300
+      COUNTDOWN_FOR_SECONDS: 60
+      MAX_UPLOAD_SIZE: 52428800
+    depends_on:
```

Comment:
The MUI section in the docker-compose file in the MUI module currently looks very different to this and only depends on info and localstack. Is everything here required?

```
  management:
    image: quay.io/ukhomeofficedigital/hocs-management-ui
    # image: hocs-frontend-test
    ports:
    - 8080:8080
    networks:
    - hocs-network
    environment:
      USE_CLIENTSIDE: 'true'
      INFO_SERVICE: http://info:8080
    depends_on:
    - info
    - localstack
```

Reply:
It looks like that is very out of date and would be worth writing another ticket to go through all of the various repos and fixing these up. 

MUI currently has 3 clients that it can talk directly too: 
1. Info
2. Casework
3. Documents 

Workflow is not actually a client of MUI and thus can be removed.

Localstack is required (and thus aws_cli - which needs adding too) for the buckets are required for templates, standard-lines etc.

The other passed through environment variables are used internally to MUI but have defaults (maybe we should be looking at whether these defaults should be altered rather than us just overwriting here. 

------

I will add aws_cli to the depends_on as that appears to be required. 
I will remove the environment variables that are no longer required as we have defaults 
I will create two tickets:
- one to review and update service level docker-compose files
- one to review the actual config we use within MUI

Reply:
Nice one thanks @LiamMacP :)